src/components/Focus/tests/Focus.test.tsx

```diff
@@ -45,14 +45,9 @@ describe('<Focus />', () => {
 
 function FocusTestWrapper({children, ...props}: Omit<FocusProps, 'root'>) {
   const root = useRef<HTMLDivElement>(null);
-  const [, setMount] = useState(false);
-
-  useEffect(() => {
```

Comment:
For testing, this was a false positive.  Forcing the component to remount would cause that stale ref to be properly set and focused.  

Removing this in the mock component with the fix makes the test work matching the component's behavior.